src/arch/riscv/vm.c

```diff
@@ -21,7 +21,7 @@ void vm_arch_init(struct vm *vm, const struct vm_config *config)
 
     CSRW(CSR_HGATP, hgatp);
 
-    vplic_init(vm, platform.arch.plic_base);
+    vxplic_init(vm, platform.arch.plic_base);
```

Comment:
I'm thinking maybe this function's second parameter should be more generic than a single address. My main reasoning is that when we support AIA these parameters might be more complex. Something on the lines of a 'struct irqc_desc*' ? Maybe I'm wrong and we can find most other AIA parameters by dynamically probing it. What do you think?